src/steps/03-Background/CurrentEnvironment/EnvironmentSummary.vue

```diff
@@ -59,9 +59,7 @@
             <template v-slot:item.location="{ item }">
               <span
                  v-html="item.location"
-                :class="[{'text-error font-weight-500': !item.isValid }]">
-                {item}
```

Comment:
According to my linter, {item} will never be set, since the v-html overrides it. Idk enough about Vue yet to fully confirm this. So I'm not totally sure it's no longer needed. I was 100% blindly trusting Vue's linting rules, it's an error level lint rule. I can make a note to go back and check that everything renders the same after build / serve works. I'm down to handle this however though.

Reply:
David's right.  Let's remove the `{item}` as the `v-html will override element children.vue(54)`